pylds/tools.py

```diff
@@ -74,12 +74,23 @@ def draw_lagrangian_descriptor(LD, LD_type, grid_parameters, tau, p_value, norm
         string_title = ''
         print('Incorrect "LD_type". Valid options: forward, backward, total. Plot will appear without title')
     
-    ax.set_title(string_title, fontsize=12)
-    ax.set_xlabel('$x$', fontsize=18)
-    ax.set_ylabel('$y$', fontsize=18)
+    fig.suptitle(string_title, fontsize=14, y=1.04)
+    ax0.set_title('LD values')
+    ax0.set_xlabel('$x$')
+    ax0.set_ylabel('$y$')
+    
+    fig.colorbar(con0, ax=ax0, ticks=np.linspace(np.nanmin(LD),np.nanmax(LD),11),format='%.2f')
+    
+    gx,gy = np.gradient(LD, 0.05, 0.05)
+    scalar = np.sqrt(gx**2 + gy**2)
+    Z = scalar/scalar.max()
+    
+    con1 = ax1.contourf(points_x,points_y,Z,cmap='Reds',levels=200)
```

Comment:
As mentioned in issue #14 
A colour convention should be followed. Reds for unstable ones, and Blues for stable ones. 
I think we discussed this previously, right @VikJGG ?

Reply:
The function plots the magnitude of the gradient for a given array LD. Forward and backward LD gradients can have different colours, but there is no way to distiguish stable and unstable in a forward+backward LD array. I agree it would be nice, but this requires an overhaul of the plotting function. We did not discuss how to do it.